output-comparator/src/main/java/org/mobilitydata/gtfsvalidator/outputcomparator/io/CorruptedSourcesCollector.java

```diff
@@ -14,15 +13,19 @@ public class CorruptedSourcesCollector {
   private final float percentCorruptedSourcesThreshold;
 
   private int sourceIdCount = 0;
-  private final List<String> corruptedSourceIds = new ArrayList<>();
+  private final List<CorruptedSourceDetail> corruptedSourceDetails = new ArrayList<>();
```

Comment:
A very small detail, but I think a TreeMap instead of a List would have done the job nicely here.